src/redux/actions/projects/saveProject.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-param-reassign */
+import fetchAPI from '../../../utils/fetchAPI';
+import pushNotificationMessage from '../notifications';
+import messages from '../../../components/notification/messages';
+
+const saveProject = (projectUuid) => async (dispatch, getState) => {
+  const project = getState().projects[projectUuid];
+
+  try {
+    await fetchAPI(
+      `/v1/projects/${projectUuid}`,
+      {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(project),
+      },
+    );
+  } catch (e) {
+    dispatch(pushNotificationMessage('error', messages.connectionError, 5));
```

Comment:
you are missing an action type for when there is an error with project creation and dispatching that action type